ios/push-notifications.md

```diff
@@ -175,110 +225,35 @@ Make the following changes to the configuration file. The values are available i
-
-1. In Xcode Project Navigator, choose your app name at the top, choose your app name under **Targets**, choose the **Capabilities** tab, and then turn on **Push Notifications**.
+The following steps show how to send push notifications targeted for your app.
 
-    ![Image of turning on Push Notifications capabilities in Xcode]({{image_base}}/xcode-turn-on-push-notification.png)
+1. Go to https://developer.apple.com/account/ 
 
-1. Configure the app to run in the **Release** profile instead of the default **Debug** profile. Perform the following steps to get a notification to the device:
+2. Under "Certificates, Identifiers & Profiles", on the left side click on "Keys", click +, type in a name like "Push Notification Key", check off Apple Push Notification Service (APNs). Register and download the file. It should be in the format of `AuthKey_<Key ID>.p8`
 
-    1. For your app target, go to the **General** tab of project configuration and make sure **Automatically Manage Signing** check box is not selected.
+2. Go to your Membership details page to get the TeamID
 
-    1. In the **Signing (Release)** section, choose the production provisioning profile you created on Apple developer console. For testing push notifications on a device, you will need an [Ad Hoc Provisioning Profile](https://help.apple.com/xcode/mac/current/#/dev4335bfd3d) configured with a Production AppStore and Ad Hoc certificate, and with the device(s) to be used for testing.
+3. Open the AWS Pinpoint console with `amplify console analytics`
 
-    1. In the top left corner of Xcode (where your app name is displayed next to the current build target device), choose on your app name and then select **Edit Scheme**, and then set **Build configuration** to **Release**
+4. Go to Settings, enable push notifications for APNS, select Key certificate, and add the Team ID, Key ID, the unique bundle identifier of your app, and select the `AuthKey_<Key ID>.p8` file
 
-        **Note**: Run your app on an iPhone device to test. Push notifications are not supported on simulators.
+5. Go to Campaign, provide a campaign name, and click next.
 
-    1. Xcode will give an error that it could not run the app, this is due to production profile apps not being allowed to debug. Click **Ok** and launch the app directly from the device.
+6. In the segement section, select `Create a new segment` and you should see 1 device as a targeted endpoint, which is the app we are running on the device. Choose this option and then choose **Next Step**.
 
-    1. When prompted, chose to allow notifications for the device.
+7. Provide text for a sample title and body for push notification, click on Test message. Enter the `endpointID` retrieved from the app in the earlier step to send a push notification to the device. 
```

Comment:
cant complete cuz dont know how to get `endpointId`

Reply:
nit: Click `Create a new push notification` and then ....Provide text for a sample....

Reply:
alright logs finally printed endpointId after a few minutes so tested this but the response I get looks like its not a normal PN. 
![IMG_0698](https://user-images.githubusercontent.com/8533749/73104819-31431b80-3eac-11ea-89bb-3215f6a6f21c.PNG)

Reply:
updating to say " enter the device token or endpoint ID retrieve from the app..."

Reply:
the modal content is because pinpoint will wrap the message that you send from AWS Pinpoint console with some data that contains campaign details. i would be happy as long as you were able to receive the alert pop up when the app is in foreground. I think I can add some more details around this